docs/manage/fleet/system-settings.md

```diff
@@ -0,0 +1,199 @@
+Custom repos installed on the system at the time the setting is enabled will be included.
+
+To set automatic upgrades, add the `upgrades` field to the `agent-syscfg`'s `attributes` object.
+You may need to add the `agent-syscfg` object to the `agent` object if it doesn't already exist.
+
+```json
+"agent": {
+    "agent-syscfg": {
+        "release_channel": "stable",
+        "attributes": {
+            "upgrades": {
+                "type": "security"
+            }
+        }
+    }
+}
+```
+
+## Configure networks
+
+By default, your machine can connect to networks added at the operating system level, for example, directly in NetworkManager.
+
+For an already-online device, you can add new WiFi networks by updating the `"agent"` value in the machine's JSON configuration.
+This is primarily useful for a machine that moves between different networks, so the machine can automatically connect when moved between locations.
+
+To add networks, add the `networks` field to the `agent-provisioning`'s `attributes` object and set `"roaming_mode": true`.
+You may need to add the `agent-provisioning` object to the `agent` object if it doesn't already exist.
+
+{{< alert title="Note" color="note" >}}
+If you are using the Viam app to add networks to a machine’s configuration, the machine will need to be connected to the internet to retrieve the configuration information containing the network credentials before it can use them.
+{{< /alert >}}
+
+The following configuration defines the connection information and credentials for two WiFi networks named `fallbackNetOne` and `fallbackNetTwo`.
+`viam-agent` will try to connect to `fallbackNetOne` first, since its priority is highest.
+If the `fallbackNetOne` is not available or the machine can connect but internet is not available, `viam-agent` will then attempt to connect to `fallbackNetTwo`.
+
+```json
+"agent": {
+    "agent-provisioning": {
+        ...
+        "attributes": {
+            "roaming_mode": true,
+            "networks": [
+                {
+                "type": "wifi",
+                "ssid": "fallbackNetOne",
+                "psk": "myFirstPassword",
+                "priority": 30
+                },
+                {
+                "type": "wifi",
+                "ssid": "fallbackNetTwo",
+                "psk": "mySecondPassword",
+                "priority": 10
+                }
+            ]
+        }
+    }
+}
+```
+
+For complete reference information, see [viam-agent](/configure/agent/#networks).
+
+### Example 2
+
+
+
+## Configure operating system logging
+
+By default, the operating system defaults determine the disk space that `journald` uses to persist logs.
+To adjust these settings update the `"agent"` value in the machine's JSON configuration.
+
+For complete reference information, see [viam-agent](/configure/agent/#agent-syscfg).
+
```

Comment:
Should mention that if NOT set, both of these fields are set to 512MB by default (unless disable is set.)

Reply:
Does that mean this sentence is wrong or that the moment we set a logging value even if it is empty the value 512 is used? 

```
By default, the operating system defaults determine the disk space that `journald` uses to persist **logs.**
```

Reply:
Correct. We enforce 512MB by default (per request from Eliot) when this is not set, which is typically smaller than most OS defaults. That's why I added the "disable" option to avoid this for people that don't want it changed.